docs-markdown/markdownlint-custom-rules/securelinks.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+"use strict";
+
+const common = require("./common");
+const detailStrings = require("./strings");
+
+module.exports = {
+  "names": ["secure-links"],
+  "description": "All links to Microsoft properties should be secure",
+  "tags": ["compliance"],
+  "function": function rule(params, onError) {
+    params.tokens.filter(function filterToken(token) {
+      return token.type === "inline";
+    }).forEach(function forToken(inline) {
+      inline.children.filter(function filterChild(child) {
+        return child.type === "link_open";
+      }).forEach(function forToken(link) {
+        if (link.attrs[0] && link.attrs[0][0] == "href") {
+          const linkPattern = /^(http:\/\/www\.|http:\/\/)([^\/]*)\/.*$/;
+
+          const href = link.attrs[0][1];
+          if (href.startsWith("http://")) {
```

Comment:
I don't think it's a big deal but could case be an issue?  For example, [Go get Visual Studio](HtTps://www.visualstudio.com/foo/test.txt) does not get picked up for comparison.

Reply:
added an appropriate test and *yes* it was a problem, updated code to handle